release-prep

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# bundle install
+docker run -it --rm \
+  -v $(pwd):/app \
+  ruby:2.7-slim-bullseye \
+  /bin/bash -c 'apt-get update -qq && apt-get install -y --no-install-recommends build-essential git make && cd /app && gem install bundler && bundle install --jobs 3; echo "LOCK_FILE_UPDATE_EXIT_CODE=$?"'
```

Comment:
Trivial: Would prefer to see this script written to the local directory then executed than passed to 'bash -c'

Makes debugging and forensics easier plus relieves of having to string together a bunch of '&&' clauses

Reply:
No need for separate file. I would do it something like this:

```
#!/usr/bin/env bash 

cat > docker-go <<-DOCKER_GO
#!/usr/bin/env bash -eux

apt-get --quiet --quiet update
apt-get --yes install --no-install-recommends build-essential git make
cd /app
gem install bundler
bundle install --jobs 3
echo "LOCK_FILE_UPDATE_EXIT_CODE=$?"'
DOCKER_GO

chmod 755 docker-go

docker run --interactive --tty --rm --volume=$(pwd):/app ruby:2.7-slim-bullseye /app/docker-go
```

Reply:
Ah okay, I'll create a follow up ticket then so we can update it across all repos for consistency